src/ray/util/tests/pipe_logger_test.cc

```diff
@@ -178,4 +202,35 @@ TEST(PipeLoggerTestWithTee, RotatedRedirectionWithTee) {
 
 }  // namespace ray
 
+#elif defined(_WIN32)
+
+#include <windows.h>
+
+#include "ray/util/tests/windows_test_utils.h"
+
+namespace ray {
+
+TEST(PipeLoggerTestWithTee, RedirectionWithNoTeeAndRotation) {
+  const std::string test_file_path = absl::StrFormat("%s.out", GenerateUUIDV4());
+
+  StreamRedirectionOption logging_option{};
+  logging_option.file_path = test_file_path;
+
+  auto log_token = CreateRedirectionFileHandle(logging_option);
```

Comment:
why it's called log_token not redirection_file_handle?

Reply:
renamed for option and token.